packages/components/package.json

```diff
@@ -68,7 +68,6 @@
 		"gradient-parser": "^0.1.5",
 		"highlight-words-core": "^1.2.2",
 		"is-plain-object": "^5.0.0",
-		"lodash": "^4.17.21",
```

Comment:
Lodash is no longer bundled for the web build since it was only used for mobile components, but in any case it's nice to see it gone from one of our largest packages.

Reply:
Is this the reason why there are no bundle size differences for this PR?

Reply:
Yes 😉 You can see the bundle size reduction and Lodash removal when we removed the last non-mobile bits:

* Last bundle that contains Lodash: https://bundlephobia.com/package/@wordpress/components@23.5.0
* First bundle that no longer contains Lodash: https://bundlephobia.com/package/@wordpress/components@23.6.0